src/index.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To-Do list</title>
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"
+      integrity="sha512-SfTiTlX6kk+qitfevl/7LibUOeJWlt9rbyDn92a1DqWOw9vWG2MFoays0sgObmWazO5BQPiFucnnEAjpAB+/Sw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+</head>
+<body>
+    <section class="box">
+        <header class="top"><h1>Today's To Do </h1></header>
+        <div class="your">
+            <input class="add-input" placeholder="Add to your list..." title="Add list">
+            <button type="submit" class="add">
+                <i class="fa fa-arrow-left" aria-hidden="true"></i>
+            </button>
+        </div>
+        <div class="input"></div>
```

Comment:
- [ ] Kindly add the marked items to match the given basic design. You don't need to add any functionality, now.

![image](https://user-images.githubusercontent.com/60083437/228687804-f5867a4e-8f5d-4157-8212-b0dae4884f3d.png)